doc/_templates/layout.html

```diff
@@ -167,91 +31,55 @@ <h3>{{ _('Navigation') }}</h3>
-    <img src="{{pathto("_static/logo2.png", 1) }}" height="125px" border="0" alt="matplotlib"/></a>
-{%- endif %}
-
-<!-- The "Fork me on github" ribbon -->
-<div id="forkongithub"><a href="https://github.com/matplotlib/matplotlib">Fork me on GitHub</a></div>
-</div>
-
-{% endblock %}
-
-{%- block relbar2 %}
-
-{{ relbar() }}
-
-{% endblock %}
-
-
-
-{%- block content %}
-  {%- block sidebar1 %} {# possible location for sidebar #} {% endblock %}
-
-  {%- block sidebar2 %}{{ sidebar() }}{% endblock %}
-    <div class="document">
-  {%- block document %}
-      <div class="documentwrapper">
-      {%- if render_sidebar %}
-        <div class="bodywrapper">
-      {%- endif %}
-          <div class="body">
-            {% block body %} {% endblock %}
-          </div>
-      {%- if render_sidebar %}
-        </div>
-      {%- endif %}
-      </div>
-  {%- endblock %}
-
-      <div class="clearer"></div>
+    {%- if builder in ('htmlhelp', 'devhelp', 'latex') %}
+    <a href="{{ pathto('index') }}">
+        <div style="float: left; position: absolute; width: 496px; bottom: 0; padding-bottom: 24px"><span style="float: right; color: #789; background: white">Version {{ version|e }}</span></div>
+        <img src="{{pathto("_static/logo2.png", 1) }}" height="125px" border="0" alt="matplotlib"/></a>
+    {%- else %}
+    <a href="{{ pathto('index') }}">
+        <div style="float: left; position: absolute; width: 496px; bottom: 0; padding-bottom: 24px"><span style="float: right; color: #789; background: white">Version {{ version|e }}</span></div>
+        <img src="{{pathto("_static/logo2.png", 1) }}" height="125px" border="0" alt="matplotlib"/></a>
+    {%- endif %}
+    
+    <!-- The "Fork me on github" ribbon -->
+    <div id="forkongithub"><a href="https://github.com/matplotlib/matplotlib">Fork me on GitHub</a></div>
     </div>
 {%- endblock %}
 
+{%- block sidebar1 %}{{ sidebar() }}{% endblock %}
+{%- block sidebar2 %}{% endblock %}
+
 {%- block footer %}
+<footer>
     <div class="footer">
-    {%- if show_copyright %}
-      {%- if hasdoc('copyright') %}
-        {% trans path=pathto('copyright'), copyright=copyright|e %}&copy; <a
+{%- if show_copyright %}
+{%- if hasdoc('copyright') %}
+    {% trans path=pathto('copyright'), copyright=copyright|e %}&copy; <a
 href="{{ path }}">Copyright</a> {{ copyright }}.{% endtrans %}
-      {%- else %}
-        {% trans copyright=copyright|e %}&copy; Copyright {{ copyright }}.{%
+{%- else %}
+    {% trans copyright=copyright|e %}&copy; Copyright {{ copyright }}.{%
 endtrans %}
-      {%- endif %}
-    {%- endif %}
-        <br />
-    {%- if last_updated %}
-      {% trans last_updated=last_updated|e %}Last updated on {{ last_updated }}.{% endtrans %}
-    {%- endif %}
-    {%- if show_sphinx %}
-	{% trans sphinx_version=sphinx_version|e %}Created using
-	<a href="http://sphinx-doc.org/">Sphinx</a> {{ sphinx_version }}.{% endtrans %}
-    {%- endif %}
-    {%- if sha %}
-	Doc version {{ sha }}.
```

Comment:
I think, this should stay as well.

Reply:
We need this because we build the docs from a different branch than the release.